devlib/platform/gem5.py

```diff
@@ -127,6 +129,11 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Enable power modelling if required
+            if self.power_models_dir:
```

Comment:
The gem5 configuration scripts are written by the user, hence why they are so diverse and non-standardised. I think Quentin is right and if these parameters are exposed here, the user might wrongly assume devlib sets up certain power related setting for gem5.